reference/5.1/Microsoft.PowerShell.Utility/Measure-Command.md

```diff
@@ -155,8 +181,9 @@ Accept wildcard characters: False
 
 ### -InputObject
 
-Objects bound to the `InputObject` parameter are optional input for the `ScriptBlock` passed to the `Expression` parameter.
-Inside the `ScriptBlock`, `$_` can be used to reference the current object in the pipeline.
+Objects bound to the `InputObject` parameter are optional input for the `ScriptBlock` passed to the
```

Comment:
`InputObject` should not be code fenced. Should be bold **InputObject** since it isn't `-InputObject`. Same for `Expression`

Reply:
Fixed this and others.